packages/alfa-tree/src/tree.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference lib="dom" />
+
```

Comment:
```suggestion
```
Is this a leftover from a `console.log`?

Reply:
No, it's required for `crypto`. Is there a better way to include it?